### PR TITLE
Update wdb.q

### PR DESCRIPTION
**Pull request by jreillyDI**
Added sync flushing to wdb in case of eodwaittime = 0

### CODE REVIEW

File: code/processes/wdb.q [hunk 1]
```diff
@@ -256,8 +259,9 @@ doreload:{[pt]
     /-inform gateway of reload start
     informgateway(`reloadstart;`);
     getprocs[;pt] each reloadorder;
-    if[eodwaittime>0;
+    $[eodwaittime>0;
         .timer.one[timeouttime::.proc.cp[]+eodwaittime;(value;".wdb.flushend[]");"release all hdbs and rdbs as timer has expired";0b];
+        .timer.one[.proc.cp[];(value;".wdb.flushend[]");"flush all hdbs and rdbs"; 0b]
```
Comment (jonathonmcmurray) [inline code/processes/wdb.q:264]: do we need to do this as a timer? can we not just call it directly?

```suggestion
        .wdb.flushend[]
```

File: code/processes/wdb.q [hunk 2]
```diff
@@ -240,7 +240,10 @@ handler:{
 /- notify the gateway that we are done
 flushend:{
     if[not @[value;`.wdb.reloadcomplete;0b];
-     @[{neg[x]"";neg[x][]};;()] each key reloadsummary;
+     $[eodwaittime>0;
+      @[{neg[x]"";neg[x][]};;()] each key reloadsummary;
+      @[{x"";x[]};;()] each key reloadsummary
```
Comment (jonathonmcmurray) [inline code/processes/wdb.q:245]: is this needed? if we sent sync requests, we shouldn't have to flush the handles?

File: code/processes/wdb.q (hunk 1 above)
Comment (jreillyDI) [reply code/processes/wdb.q:264]: Apologies, I think I had misunderstood the purpose of having the timer function here

File: code/processes/wdb.q (hunk 2 above)
Comment (jreillyDI) [reply code/processes/wdb.q:245]: Hadn't thought of that. Apologies.

**Review by jonathonmcmurray — APPROVED** [review]
LGTM

**Pull request merged**